relentless/optimize/method.py

```diff
@@ -339,14 +469,38 @@ def optimize(self, objective):
         iter_num = 0
         cur_res = objective.compute()
         while not self.has_converged(cur_res) and iter_num < self.max_iter:
+            grad_y = cur_res.gradient
+            if np.isscalar(self.scale):
+                grad_y *= self.scale**2
```

Comment:
```suggestion
                grad_y *= self.scale
```

Just one scale factor in the gradient.